Fixes incorrect Version Comparison

lineargraph opened the pull request:
Now ClassFileVersion.supports returns the correct result for versions with lower major version but higher minor version

lineargraph committed:
Fixes incorrect Version Comparison

Now ClassFileVersion.supports returns the correct result for versions with lower major version but higher minor version

Stupremee commented on tests/class_file.rs:
```diff
@@ -1,11 +1,17 @@
 use crate::class_file::ClassFileVersion;
```
Please remove this line since it is unnecessary.

Stupremee commented on tests/class_file.rs:
```diff
@@ -1,11 +1,17 @@
     assert!(first.supports(second));
+    assert!(first.supports(third));
+    assert!(second.supports(third));
     assert_ne!(second.supports(first));
```
`assert_ne!` won't work here, because it checks if two objects are **not** the same. You have to replace it with `assert!(!condition)`

romangraef replied:
I was wondering, because i think this code was already there before. Anyway, I'll change all those.

lineargraph committed:
Requested Changes: assert_ne! -> assert!(! ... )

Changes `assert_ne!()` to `assert!(!)` as per Request. Also removed
unnecessary `use`.
